src/Test.FunctionalTests.BinSkim.Driver/Test.FunctionalTests.BinSkim.Driver.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., build.netcore.props))\build.netcore.props" />
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreVersion);</TargetFrameworks>
+	<TargetFrameworks>$(NetCoreVersion);$(NetSixVersion)</TargetFrameworks>
```

Comment:
>NetSixVersion [](http://example.com/codeflow?start=39&length=13)

Suggest just updating the value of NetCoreVersion rather than creating a new property.